tasks/package.js

```diff
@@ -96,9 +168,12 @@ module.exports = function(grunt) {
       }
     });
 
-    grunt.config.set('clean.packages', [destPath]);
-
-    tasks.push('clean:packages');
+    if (useRsync) {
+      tasks.push('mkdir:package');
+      tasks.push('shell:rsync');
+    } else {
+      tasks.push('copy:source');
+    }
     tasks.push('copy:package');
```

Comment:
Yeah, I even thought of "copy:projFiles" to be really specific.  If we call it "package-extras" then having the config be "projFiles" might be confusing too.